README.md

```diff
@@ -0,0 +1,28 @@
+# TiKV RFCs
+
+Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow.
```

Comment:
I see that this part comes directly from the Raft RFC README. Can we rephrase this part and makes it our own words? Maybe something like:
The "RFC" (request for comments) process is to provide a consistent and controlled path to propose major changes or enhancements to TiKV to keep the stakeholders on the same page. An RFC can work as a design document to describe the high-level description and serves as a form of communication among stakeholders.

Reply:
@queenypingcap this is not for Raft RFC only, you can see many projects' RFC readme, they use the same words here.

Reply:
E.g, Rust also uses this.

Reply:
I see. But does that mean we are doing the same?

Reply:
I think keeping the same is ok

/cc @Hoverbear